webapp/sources/rudder/rudder-web/src/main/javascript/rudder/rudder-datatable.js

```diff
@@ -43,6 +43,15 @@ var inventories = {};
 var recentGraphs = {};
 var nodeIds = undefined;
 var scoreList = [];
+
+$.ajax({
+  type: 'GET',
+  url: "/rudder/secure/api/scores/list",
+  success: function (data) {
+    scoreList = data.data
+  },
+})
+
```

Comment:
We still have a race condition with this block, that executes asynchronously (`$.ajax` is fire-and-forget).
When it finished BEFORE the execution of the code that looks into localStorage for score columns, it works normally. But it often finishes AFTER, and the list is `[]`, so we are still displaying `system-updates Score` instead of `System updates Score` (because there is a "guard" to just display the `id` when we don't know the `name` of the score...)